docker-compose/cse-valid-d2cc/config/data-bridge-d2cc-crac-export-corner.yml

```diff
@@ -0,0 +1,25 @@
+data-bridge:
+  zone-id: "Europe/Paris"
+  target-process: CSE_VALID_D2CC
+  file-type: CRAC
+  file-regex: "(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_CRAC_Validation_Transit_FR[0-9].(xml|XML)"
+  time-validity: hourly
+  remote-file-regex:
+    - "CRAC_2D_CSE_Validation_Transit_[0-9]{8}V[0-9]{2}.zip"
+    - "(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_2D[1-7]_CRAC_Validation_Transit_FR[0-9].(xml|XML)"
```

Comment:
Actually here we don't have to catch the values year, month, etc... so you can just put [0-9]{8}_[0-9]{4}_etc